### ISSUE TITLE
newman failure details getting truncated

### ISSUE THREAD

**Issue by bobbyi** [postmanlabs/newman#2293]
When running newman on my CI server, if there are many test failures, the list of individual failures (at the end of the CLI output, after the table with "total run duration", etc.) does not get output completely. I see roughly the first 20 failures, and then further output is cut off. 

The point where it is cut off often ends in the middle of a line or word, so it seems there is some sort of issue where an output buffer isn't being completed flushed or something of that nature as opposed to there genuinely only being that many failures getting detailed.

If I run newman in the exact same way on my laptop, I see the full list of details for all failures. To be clear, this is invoking the newman CLI with the CLI reporter, not embedding it as a library.

I am confident that this isn't an issue with the CI server itself failing to show the full output of the newman command it invoked. I have tried changing it from running newman directly to running a shell script which runs newman and saves its output to a file, and I still see the output truncated in the file. Also, when I have it running newman via "npm run" , I see the npm messages about newman having failed after the truncated output.

I've tried upgrading my newman version to 4.6.0 and my node version to 12.16.1 and that did not resolve the issue. The server machine is 64-bit linux, although I would have to do more research if more details are needed about the specific environment.

Is there any known case where output would be truncated in this way, or any possible workarounds I could try to get the full output?

**Comment by DannyDainton** [comment]
Hey @bobbyi 

I'm going to close this issue out for now but if you're still having problems, we can open it again.

**Comment by bobbyi** [comment]
@DannyDainton Sorry, I've been busy this week and haven't had a chance to try it with the latest version. I will do that soon.

**Comment by DannyDainton** [comment]
That's cool - Once you get a chance to look again, if you could give us an example screenshot, we'll be able to see what's going on.

Hopefully, get this all sort if you're still seeing the problem.

Enjoy the weekend! 🏆

**Comment by mheine** [comment]
@DannyDainton Sorry to dig up an old ticket, but we've recently encountered the same issue as well. We're running `newman 4.5.7` (and updating to `5.2.2` does not appear to help) using npm and running it as the embedded library (as opposed to the original issue which uses the CLI). We're using `Jenkins` and `docker-compose`, and running `node 5.2.2`.

We're also encountering the same problems with the output being truncated on our CI server, but everything is running to completion locally. In addition, this also seems to occur even when there are no failures; the output is truncated but the tests themselves appear to have passed.

I've attached a screenshot to show the truncated output (with a few details regarding our suite blurred out), but it's hard to say how much this will help.

![2021-01-25-122456_2043x1437_scrot](https://user-images.githubusercontent.com/2301054/105702950-d0aa4c80-5f0c-11eb-88c4-e9ac2dabcdbc.png)
